content/java/concepts/math-methods/terms/sqrt/sqrt.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.sqrt()'
+Description: 'Returns the square root of a double value passed to it as an argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Sqrt'
```

Comment:
```suggestion
  - 'Arithmetic'
  - 'Functions'
  - 'Methods'
  - 'Numbers'
```

For the meta-`Tags`, let's use what's inside the [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) file. I suggest adding `'Functions'` for SEO-purposes, as well as the `'Numbers'` tag.